src/proposals/20231011/SparkEthereum_20231011.sol

```diff
@@ -0,0 +1,176 @@
+    address public constant RETH            = 0xae78736Cd615f374D3085123A210448E74Fc6393;
+    address public constant USDC            = 0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48;
+    address public constant USDC_PRICE_FEED = 0x8fFfFfd4AfB6115b954Bd326cbe7B4BA576818f6;
+    address public constant USDT            = 0xdAC17F958D2ee523a2206206994597C13D831ec7;
+    address public constant USDT_PRICE_FEED = 0x3E7d1eAB13ad0104d2750B8863b489D65364e32D;
+
+    uint256 public constant VARIABLE_RATE   = 44790164207174267760128000; // DSR - 0.4% expressed as a yearly APR [RAY]
+
+    function _preExecute() internal override {
+        LISTING_ENGINE.POOL_CONFIGURATOR().setEModeCategory(
+            2,
+            91_00,
+            92_00,
+            101_00,
+            address(0),
+            'USD'
+        );
+    }
+
+    function newListings()
+        public pure override returns (IEngine.Listing[] memory)
+    {
+        IEngine.Listing[] memory listings = new IEngine.Listing[](1);
+
+        listings[0] = IEngine.Listing({
+            asset:              USDT,
+            assetSymbol:        'USDT',
+            priceFeed:          USDT_PRICE_FEED,
+            rateStrategyParams: Rates.RateStrategyParams({
+                optimalUsageRatio:             _bpsToRay(95_00),
+                baseVariableBorrowRate:        0,
+                variableRateSlope1:            VARIABLE_RATE,
+                variableRateSlope2:            _bpsToRay(20_00),
+                stableRateSlope1:              0,
+                stableRateSlope2:              0,
+                baseStableRateOffset:          0,
+                stableRateExcessOffset:        0,
+                optimalStableToTotalDebtRatio: 0
+            }),
+            enabledToBorrow:       EngineFlags.ENABLED,
+            stableRateModeEnabled: EngineFlags.DISABLED,
+            borrowableInIsolation: EngineFlags.DISABLED,
+            withSiloedBorrowing:   EngineFlags.ENABLED,
+            flashloanable:         EngineFlags.ENABLED,
+            ltv:                   0,
+            liqThreshold:          0,
+            liqBonus:              0,
+            reserveFactor:         5_00,
+            supplyCap:             30_000_000,
+            borrowCap:             0,
+            debtCeiling:           0,
+            liqProtocolFee:        0,
+            eModeCategory:         2
+        });
+
+        return listings;
+    }
+
+    function capsUpdates()
+        public pure override returns (IEngine.CapsUpdate[] memory)
+    {
+        IEngine.CapsUpdate[] memory capsUpdate = new IEngine.CapsUpdate[](2);
+
+        capsUpdate[0] = IEngine.CapsUpdate({
+            asset:     RETH,
+            supplyCap: 60_000,
+            borrowCap: EngineFlags.KEEP_CURRENT
+        });
+
+        capsUpdate[1] = IEngine.CapsUpdate({
+            asset:     USDC,
+            supplyCap: 60_000_000,
+            borrowCap: EngineFlags.KEEP_CURRENT
```

Comment:
I would be explicit and override whatever the previous value is since this is pretty much a new asset.

Reply:
Done.